crowd.py

```diff
@@ -51,9 +53,10 @@ def __init__(self, crowd_url, app_name, app_pass, ssl_verify=True,
     def __str__(self):
         return "Crowd Server at %s" % self.crowd_url
 
+    # don't include the password!
     def __repr__(self):
-        return "<CrowdServer('%s', '%s', '%s')>" % \
-            (self.crowd_url, self.app_name, self.app_pass)
+        return "<CrowdServer('%s', '%s', '********')>" % \
+            (self.crowd_url, self.app_name)
```

Comment:
I think instead of a static string, we can maybe mask the password string matching it's length? 

I know this gives away the length of the password but that could still be useful for users to know which password was used to configure the server.
```suggestion
        return "<CrowdServer('%s', '%s', '%s')>" % (
            self.crowd_url, self.app_name, '*' * len(self.app_pass))
```
Optionally, we could add a private method that returns the raw password (probably out of scope for this PR).